XenkoShaderExplorer/MainViewModel.cs

```diff
@@ -66,13 +66,14 @@ public MainViewModel()
         {
             try
             {
-                var xenkoDir = Environment.GetEnvironmentVariable(XenkoEnvironmentVariable) ?? FallbackBasePath;
-                var basePath = System.IO.Path.Combine(xenkoDir, "GamePackages");
+                var userDir = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
```

Comment:
You fixed the folder lookup here which is nice 👍 However, I would like to keep the ability to define the Xenko folder via the "XenkoDir" environment variable - this way you can use the Shader Explorer with a development clone of the Xenko repository, not just with the official releases. So maybe you could first check if "XenkoDir" is set and if so, use that directory instead of the automatically detected one.

Reply:
yes, agreed. any maybe also add an additional folder, like your current project so shaders from that also get added to the tree view. i'll have a look at it.